datajunction-clients/python/Makefile

```diff
@@ -5,7 +5,7 @@ lint:
 	make check
 
 test:
-	pdm run pytest --cov=datajunction --cov-report=html -vv tests/ --doctest-modules datajunction --without-integration --without-slow-integration ${PYTEST_ARGS}
```

Comment:
This runs the Python client tests in parallel

Reply:
Did you mean it will run the tests in parallel after your code change?

Reply:
Yep! Although this only works locally (on github we just have 1 core, so the parallelism will just be 1)